examples/typescript-esm/simple_transfer/transfer.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * This example shows how to use the Aptos client to create accounts, fund them, and transfer between them.
+ */
+
+import {
+  Account,
+  AccountAddress,
+  Aptos,
+  AptosConfig,
+  Network,
+  SigningScheme,
+  StructTag,
+  TypeTagStruct,
+  U64,
+} from "aptos";
+
+const APTOS_COIN = "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>";
+const ALICE_INITIAL_BALANCE = 100_000_000;
+const BOB_INITIAL_BALANCE = 100;
+const TRANSFER_AMOUNT = 0;
+
+/**
+ * Prints the balance of an account
+ * @param aptos
+ * @param name
+ * @param address
+ * @returns {Promise<*>}
+ *
+ */
+const balance = async (aptos: Aptos, name: string, address: AccountAddress) => {
+  let resource = await aptos.getAccountResource({ accountAddress: address.toUint8Array(), resourceType: APTOS_COIN });
+  let amount = Number((resource.data as { coin: { value: string } }).coin.value);
```

Comment:
och

Reply:
This is definitely a user experience we can fix.